salt/metalk8s/defaults.yaml

```diff
@@ -6,3 +6,13 @@ repo:
     version: latest
   kubernetes:
     version: latest
```

Comment:
Consider adding an empty line here, to lay out various 'sections' of this configuration.

Reply:
Will do